tesseract/tesseract/include/tesseract/tesseract_init_info.h

```diff
@@ -63,8 +65,9 @@ struct TesseractInitInfo
 
   TODO: When SceneGraph is given a clone method make this a copy of the initial state rather than a pointer*/
   tesseract_scene_graph::SceneGraph::Ptr scene_graph;
+
   /** @brief Used when InitType is SCENE_GRAPH_SRDF_MODEL*/
-  tesseract_scene_graph::SRDFModel::ConstPtr srdf_model_;
+  tesseract_scene_graph::SRDFModel::Ptr srdf_model;
```

Comment:
Why is this non const? The init info shouldn't change.

Reply:
The tesseract object now needs a non const srdf because it can now be modified and saved if needed. You can programmatically add group states, group tcp's, kinematics objects, etc. I could change it to a unique pointer so it shows that it takes ownership. This is primarily used by the Tesseract Setup Wizard.

Reply:
Fixed.

Reply:
fixed